commit 7fd55a02a426ffff378e0acceff4a381bcbbfca0
Merge: 5e4e84f1124a 1c53a1ae3612
Author: Paolo Bonzini <pbonzini@redhat.com>
Date:   Fri Jan 7 10:42:19 2022 -0500

    Merge tag 'kvmarm-5.17' of git://git.kernel.org/pub/scm/linux/kernel/git/kvmarm/kvmarm into HEAD
    
    KVM/arm64 updates for Linux 5.16
    
    - Simplification of the 'vcpu first run' by integrating it into
      KVM's 'pid change' flow
    
    - Refactoring of the FP and SVE state tracking, also leading to
      a simpler state and less shared data between EL1 and EL2 in
      the nVHE case
    
    - Tidy up the header file usage for the nvhe hyp object
    
    - New HYP unsharing mechanism, finally allowing pages to be
      unmapped from the Stage-1 EL2 page-tables
    
    - Various pKVM cleanups around refcounting and sharing
    
    - A couple of vgic fixes for bugs that would trigger once
      the vcpu xarray rework is merged, but not sooner
    
    - Add minimal support for ARMv8.7's PMU extension
    
    - Rework kvm_pgtable initialisation ahead of the NV work
    
    - New selftest for IRQ injection
    
    - Teach selftests about the lack of default IPA space and
      page sizes
    
    - Expand sysreg selftest to deal with Pointer Authentication
    
    - The usual bunch of cleanups and doc update

diff --cc arch/arm64/kvm/Makefile
index 04a53f71a6b6,8ca8cf6f5619..39b11a4f9063
--- a/arch/arm64/kvm/Makefile
+++ b/arch/arm64/kvm/Makefile
@@@ -10,10 -10,12 +10,10 @@@ include $(srctree)/virt/kvm/Makefile.kv
  obj-$(CONFIG_KVM) += kvm.o
  obj-$(CONFIG_KVM) += hyp/
  
 -kvm-y := $(KVM)/kvm_main.o $(KVM)/coalesced_mmio.o $(KVM)/eventfd.o \
 -	 $(KVM)/vfio.o $(KVM)/irqchip.o $(KVM)/binary_stats.o \
 -	 arm.o mmu.o mmio.o psci.o perf.o hypercalls.o pvtime.o \
 +kvm-y += arm.o mmu.o mmio.o psci.o perf.o hypercalls.o pvtime.o \
  	 inject_fault.o va_layout.o handle_exit.o \
  	 guest.o debug.o reset.o sys_regs.o \
- 	 vgic-sys-reg-v3.o fpsimd.o pmu.o \
+ 	 vgic-sys-reg-v3.o fpsimd.o pmu.o pkvm.o \
  	 arch_timer.o trng.o\
  	 vgic/vgic.o vgic/vgic-init.o \
  	 vgic/vgic-irqfd.o vgic/vgic-v2.o \